tico/serialize/operators/op_avg_pool2d.py

```diff
@@ -26,16 +27,79 @@
 from tico.serialize.operators.node_visitor import NodeVisitor, register_node_visitor
 from tico.serialize.operators.utils import create_builtin_operator, get_op_index
 from tico.utils.define import define_pad_node
+from tico.utils.errors import NotYetSupportedError
 from tico.utils.validate_args_kwargs import AvgPool2dArgs
 
 
 @register_node_visitor
 class AvgPool2DVisitor(NodeVisitor):
+    """
+    This class defines how to serialize AvgPool2D operation into Circle IR.
+
+    Torch                           | Circle
+
+    count_include_pad: True/False   | (count_include_pad): Always False
+    (padding): Always "valid"       | padding: "valid"/"same"
```

Comment:
Torch avgpool2d Op can have padding. It's not always "valid".

https://docs.pytorch.org/docs/stable/generated/torch.nn.functional.avg_pool2d.html